pkg/domain/infra/tunnel/containers.go

```diff
@@ -624,6 +624,10 @@ func (ic *ContainerEngine) ContainerExec(ctx context.Context, nameOrID string, o
 	return inspectOut.ExitCode, nil
 }
 
+func (ic *ContainerEngine) ContainerExecNoSession(ctx context.Context, nameOrID string, options entities.ExecOptions, streams define.AttachStreams) (int, error) {
+	return 0, errors.New("--no-session is not supported for the remote client")
```

Comment:
Is there a specific blocker that would prevent a remote client from supporting exec without a session? Or will it be implemented in another PR?

Reply:
Thanks, I didn't realize that.